cgo_static.go

```diff
@@ -1,13 +1,15 @@
-//go:build !duckdb_use_lib && (darwin || (linux && (amd64 || arm64)) || (freebsd && amd64))
+//go:build !duckdb_use_lib && (darwin || (linux && (amd64 || arm64)) || (freebsd && amd64) || (windows && amd64))
 
 package duckdb
 
 /*
+#cgo CPPFLAGS: -DDUCKDB_STATIC
```

Comment:
Could you update once the respective duckdb PR is merged? Then, we can also remove the patch.